include/openspace/util/spicemanager.h

```diff
@@ -993,6 +993,55 @@ class SpiceManager {
         AberrationCorrection aberrationCorrection, double ephemerisTime,
         int numberOfTerminatorPoints);
 
+    /**
+     * The structure retuned by the #lightTime method.
+     */
+    struct LightTimeResult {
+        /// Epoch of the signal at the target.
+        /// This is the time at which the signal would arrive at the target.
+        double epochAtTarget = 0.0;
+
+        /// Time between transmit and receipt of the signal.
+        /// This is the light travel time.
+        double elapsed = 0.0;
+    };
+
+    /**
+     * This method returns time of a signal recieved at a target from an \observer
+     * corrected for light travel time.
+     *
+     * \param observedTime Time at which the signal was received
+     * \param target The target who emited the signal
+     * \param observer The observer who received the signal
+     * \param direction The direction of travel ("->" or "<-")
```

Comment:
The direction parameter should instead be an `enum class` instead, for example the AberrationCorrection::Direction which already does this